.github/workflows/tests.yaml

```
@@ -0,0 +1,54 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request: null
+  workflow_dispatch: null
+  schedule:
+    - cron: 0 23 * * 4  # Run tests at 11pm UTC every Thursday.
+
+jobs:
+  tests:
+    name: tests
+    strategy:
+      matrix:
+        pyver: ["3.11"]
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v4
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.pyver }}
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+          miniforge-version: latest
+          miniforge-variant: Mambaforge
+
+      - name: Configure conda and install dependencies
+        shell: bash -l {0}
+        run: |
+          conda config --set always_yes yes
+          mamba install -q stackvana=0
```

```suggestion
          conda install -q stackvana=0
```

conda uses the mamba solver now.